### PR TITLE
case ignored regexes for high value domain and name servers

### PR DESCRIPTION
**Pull request by nimaeskandary**
Ignored case for high value domain and name server regexes

### CODE REVIEW

File: modules/api/src/main/scala/vinyldns/api/VinylDNSConfig.scala [hunk 1]
```diff
@@ -58,10 +59,11 @@ object VinylDNSConfig {
   lazy val cryptoConfig: Config = vinyldnsConfig.getConfig("crypto")
   lazy val system: ActorSystem = ActorSystem("VinylDNS", VinylDNSConfig.config)
   lazy val approvedNameServers: List[Regex] =
```
Comment (rebstar6) [inline modules/api/src/main/scala/vinyldns/api/VinylDNSConfig.scala]: need to check the thing comparing to more so than the regex provided. if we hvd `.*high-value.*` and someone tries to make the record `HIGH-VALUE.test.com`, that should fail the hvd check.

Comment (rebstar6) [reply modules/api/src/main/scala/vinyldns/api/VinylDNSConfig.scala]: `isNotHighValueFqdn` at a minimum would be impacted

Comment (nimaeskandary) [reply modules/api/src/main/scala/vinyldns/api/VinylDNSConfig.scala]: ah okay I see what you're saying

Comment (nimaeskandary) [reply modules/api/src/main/scala/vinyldns/api/VinylDNSConfig.scala]: @rebstar6 I added more validations, let me know if that covers what you were thinking

Comment (rebstar6) [reply modules/api/src/main/scala/vinyldns/api/VinylDNSConfig.scala]: ooooh i see, i was reading this wrong the 1st time

**Pull request merged**